MapboxCoreNavigation/RouteController.swift

```diff
@@ -357,7 +353,13 @@ open class RouteController: NSObject {
      
      This is a raw location received from `locationManager`. To obtain an idealized location, use the `location` property.
      */
-    var rawLocation: CLLocation?
+    var rawLocation: CLLocation? {
+        didSet {
+            guard let coordinates = routeProgress.currentLegProgress.currentStep.coordinates else { return }
+            guard let coordinate = rawLocation?.coordinate else { return }
```

Comment:
If there’s no coordinate, should `userSnapToStepDistanceFromManeuver` be set to `nil`?